<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVTests</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGraphicsView" name="graphicsView">
            <property name="dragMode">
             <enum>QGraphicsView::ScrollHandDrag</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
            <item>
             <widget class="QPushButton" name="loadImageButton">
              <property name="text">
               <string>Load Image...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="revertButton">
              <property name="text">
               <string>Revert</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,0">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="zoomInButton">
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="zoomOutButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="edgeDetect">
             <attribute name="title">
              <string>Edge Detect</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="gaborFilter">
             <attribute name="title">
              <string>Gabor Filter</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>loadImageButton</tabstop>
  <tabstop>revertButton</tabstop>
  <tabstop>zoomInButton</tabstop>
  <tabstop>zoomOutButton</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
